src/config/settings.js

```diff
@@ -57,6 +57,17 @@ export default {
         secondary: '#00BFFF',
       },
       section_divider: 'rgba(0, 0, 0, 0.25)',
+      annotations: {
```

Comment:
This is a nice little update here which enables more customization of how annotations and searches are displayed 😎